layouts/shortcodes/matchingtextbutton.html

```diff
@@ -0,0 +1,2 @@
+<script src="{{"/js/matchingtext.js" | urlize | relURL }}"></script>
```

Comment:
We can shift this script to the page footer (using `hasShortcode`) but I'm happy to leave that for a later PR.

Reply:
i'm not sure how that is done. tried search for `hasShortcode` online.

Reply:
Oops, it's `.HasShortcode`: https://godoc.org/github.com/gohugoio/hugo/hugolib#Page.HasShortcode

Reply:
https://gohugo.io/templates/shortcode-templates/#checking-for-existence

Reply:
should it look something like this?

```
{{- if .HasShortcode "matchingtextbutton" -}}
<script src="{{"/js/matchingtext.js" | urlize | relURL }}"></script>
{{- end -}}
```

also on which line to add it in?
https://github.com/kubernetes/website/blob/0625ced4668b819eb9789570d8b9f2c5f9fef4b9/layouts/partials/footer.html